cmd/install/install.go

```diff
@@ -117,6 +119,10 @@ func (o *Options) Validate() error {
 	if o.HyperShiftImage != version.HyperShiftImage && len(o.ImageRefsFile) > 0 {
 		errs = append(errs, fmt.Errorf("only one of --hypershift-image or --image-refs-file should be specified"))
 	}
+	if o.RHOBSMonitoring && os.Getenv(rhobsmonitoring.EnvironmentVariable) != "1" {
```

Comment:
so when running install from my shell I need to RHOBS_MONITORING=1 first? why?

Reply:
Because the scheme is created in an init function. By the time you get here and the env var is not set, you already have the wrong scheme.

Reply:
what's the value of the flag then? should we just use the env var?